### PR TITLE
Refactor/global input styles

### LINKED ISSUE #56: Add global style for input components
Do we need to create global style for the input, select & textarea?
Update init.styles.js with the following:

input, 
textarea, select {
	padding: var(--small-space) var(--smallest-space);
        border: 0.1rem solid var(--lightblue);
        background: var(--lightgrey);
        border-radius: 0.5rem;
}

placeholder {
       color: var(--darkgrey);
       font-size: 1rem;
}

### PR DESCRIPTION
**Pull request by catherinehoward**
Closes #56

This pull request adds global styles for the input components: TextArea, TextInput and Select.

### CODE REVIEW

**Comment by catherinehoward** [comment]
I can't get StoryBook too work at the moment. I'm getting the following error:

Unable to resolve path to module './src/theme/inputs.styles'  imp
ort/no-unresolved

It doesn't like my import statement for InputWrapper in TextArea.styles.js. Can you let me know what the correct address path is? Tried a few different things!

File: src/theme/inputs.styles.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const InputWrapper = styled.div`
+  input,
+  textarea,
+  select {
+    padding: var(--small-space) var(--smallest-space);
```
Comment (RobTF9) [inline src/theme/inputs.styles.js]: change padding to `padding: var(--small-space);` like you originally had in the TextInput component

File: src/theme/index.js [hunk 2]
```diff
@@ -11,6 +12,7 @@ const Theme = () => {
       <Palette />
       <Typography />
       <Spacings />
+      <InputWrapper />
```
Comment (RobTF9) [inline src/theme/index.js]: You don't need to import this here. You're not creating a global style.

**Review by RobTF9 — CHANGES_REQUESTED** [review]

**Comment by catherinehoward** [comment]
@RobTF9 I've made those changes if you could check again. Thanks

**Review by RobTF9 — APPROVED** [review]

**Pull request merged**